.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+Thank you for working on TiKV! Please read TiKV's [CONTRIBUTING](https://github.com/pingcap/tikv/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
+
+## What have you changed? (mandatory)
+
+Please explain **IN DETAIL** what the changes are in this PR and why they are needed:
+- Summarize your change (mandatory)
+- How does this PR work? Need a brief introduction for the changed logic (optional)
+- Describe clearly one logical change and avoid lazy messages (optional)
```

Comment:
If there are multiple logical changes this seems like a request to refactor a single pull request into multiple. A clearer wording that doesn't require that would be:

Separately describe each logical change.